webcompat/static/js/lib/issue-list.js

```diff
@@ -404,10 +404,7 @@ issueList.IssueView = Backbone.View.extend(
     // depending on what category was clicked (or if a search came in),
     // update the collection instance url property and fetch the issues.
 
-    // new is a special category that must be retrieved via the Search API,
-    // rather than the Issues API (which can return results for labels)
-      var searchAPICategories = ['new'];
-      var issuesAPICategories = ['closed', 'contactready', 'needsdiagnosis',
+      var issuesAPICategories = ['needstriage', 'closed', 'contactready', 'needsdiagnosis',
```

Comment:
nit: `needstriage` should be alphabetized in this array, please (and since you're here, you can swap the order of needsdiagnosis and needscontact).